externals/llvm-external-projects/torch-mlir-dialects/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
 if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+# This disables the setup whereby torch-mlir-dialects is
+# built as a completely standalone project, ie neither as
+# part of LLVM or as a subdirectory of torch-mlir.
```

Comment:
```suggestion
# The cmake configuration to build torch-mlir-dialects on
# its own out-of-tree has not been implemented. It can
# be built as part of LLVM or as a subdirectory of torch-mlir.
```